src/main/java/com/minecolonies/colony/WorkOrderView.java

```diff
@@ -0,0 +1,119 @@
+package com.minecolonies.colony;
+
+import io.netty.buffer.ByteBuf;
+import net.minecraftforge.fml.common.network.ByteBufUtils;
+
+/**
+ * The WorkOrderView is the client-side representation of a WorkOrders.
+ * Views contain the WorkOrder's data that is relevant to a Client, in a more client-friendly form
+ * Mutable operations on a View result in a message to the server to perform the operation
+ */
+public class WorkOrderView
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a constructor to the class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1258)